carts/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+
+class Cart(models.Model):
+    user           = models.ForeignKey('users.User', on_delete=models.CASCADE)
+    product        = models.ForeignKey('products.Product', on_delete=models.CASCADE)
+    quantity       = models.IntegerField()
+    price          = models.DecimalField(max_digits=10,decimal_places=2)
+    created_at     = models.DateTimeField(auto_now_add=True)
+    updated_at     = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = 'carts'
+
+class Order_Item(models.Model):
```

Comment:
class 명은 `camelCase`로 만들어주세요-!
```suggestion
class OrderItem(models.Model):
```